src/EA.Weee.Core/Scheme/CategoryValues.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace EA.Weee.Core.Scheme
+{
+    using System;
+    using System.Collections.Generic;
+    using EA.Weee.Core.DataReturns;
+
+    public class CategoryValues<T> : List<T> where T : CategoryValue, new()
+    {
+        public CategoryValues()
+        {
+            foreach (var category in GetEnumValues())
+            {
+                Add((T)Activator.CreateInstance(typeof(T), category));
+            }
+        }
```

Comment:
reuse the AATF one

Reply:
removed and replaced with the one in the Shared